zomes/hc_zome_trust_atom/src/trust_atom.rs

```diff
@@ -420,6 +442,22 @@ mod tests {
 
   // #[test]
   // fn test_normalize_value() {
-  //   // assert_eq!(normalize_value("0.9").unwrap(), ".900000000");
+  //   assert_eq!(normalize_value("0.9"), ".999999999");
+
+  //   // let test_data = [
+  //   //   "0"
+  //   // ];
+  //   // for item in &test_data {
+  //   //   let input: u8 = item.0;
+  //   //   let expected = item.1;
+  //   //   assert_eq!(normalize_value(value), expected);
+  //   // }
   // }
+
+  #[test]
+  fn test_gen_bucket() {
+    let test = "123456789";
```

Comment:
you need to call the function under test, something like:

```suggestion
    let test = gen_bucket();
```